parser/src/lexer.rs

```diff
@@ -557,6 +571,11 @@ where
                                 error: LexicalErrorType::StringError,
                                 location: self.get_pos(),
                             });
+                        } else if is_bytes && !c.is_ascii() {
```

Comment:
It looks like clippy wants you to merge this condition with the one above: `if c == '\n' && !triple_quoted || is_bytes && !c.is_ascii()`. (It'll probably ask you to make it single `else if` after that, too, rather than an `else { if ... { } }`